ext/extconf.rb

```diff
@@ -58,6 +58,10 @@
   $srcs = Dir['libsass/src/**/*.{c,cpp}']
 end
 
+if enable_config('strip', RbConfig::CONFIG['host_os'].downcase !~ /darwin/)
```

Comment:
Please add a comment with a link to https://github.com/sass/sassc-ruby/pull/174

Reply:
done

I would like to test it, but all tests passed on OSX (multiple versions) even without this change :-(
https://travis-ci.org/sass/sassc-ruby/builds/652242014?utm_source=github_status&utm_medium=notification